haya-hayaodeh/Homework-1/fizzbuzz/app.js

```diff
@@ -25,3 +25,19 @@
 // ##### Step 4:
 // Add an additional `else if` clause that logs the string `"fizzbuzz"` if the value being iterated over is divisible by both `3` and `5`.
 
+for (let i = 1; i < 101; i++) {
+  if (i % 3 === 0) {
+    console.log('fizz');
+    } 
+  if (i % 5 === 0) {
+    console.log('buzz');
+  }  
+    
+  if (i % 5=== 0, i%3===0) {
```

Comment:
Once you incorporate the `else if` clauses, check your output in the console and think about when this clause needs to be evaluated in order to actually run. (This is the trickiest part of this exercise!)